MiniProject1-master/transcript_section.csv

```diff
@@ -0,0 +1,11 @@
+Philip Melter,"Like  having some of this linear  like,  right.   I mean  we could put some information there  and it'd maybe be useful,  but… "
+Elizabeth Homer,"True.  I think it’s so nice and concise  the way that it is now,  with… "
+Philip Melter,Yeah.
+Elizabeth Homer,The before and after.
+Philip Melter,Shouldn’t we try to pack more info into it?
+Elizabeth Homer,Right.
+Mike Lands,"I think you should zoom in.   Like  a lot of stuff… well,  or I guess  the least I’m saying  is like  a lot of our vignettes around regionals  are all really interesting,  right.   There’s the foghorn goes off while they’re running in the middle of the woods  and there’s the  like… we had to go around the lake,  can anyone hear me on the radio?   They’re seeing someone through the scope  like… "
+Philip Melter,Yeah.  Yeah.
+Mike Lands,"There’s not  like  really… they’re really detailed,  but I don’t know if they belong  in the broad interactions map...  Or if anymore details needs to be put in. "
+Philip Melter,Okay.
+Mike Lands,"Like  I don’t know if this is what Mary had in mind,  but I think it works. "
```

Comment:
I understand the concern with data/privacy, but out of curiosity: what size of a dataset were you working with? Also, I assume this has been anonymized? (on account of, it's publicly available on Github now.)